configs/e2e/src/pages/page.ts

```diff
@@ -39,6 +39,7 @@ class Page {
     }
 
     headerElement(pageTitle = "pageTitle1"): WebdriverIO.Element {
+        this.getWidget(pageTitle).waitForDisplayed();
         return this.getWidget(pageTitle);
```

Comment:
You can save `getWidget` result in a variable iso doing it twice.

Reply:
Done